modules/ddl-and-loading/partials/load-statement.adoc

```diff
@@ -128,7 +140,9 @@ Arithmetic Operators:: `+`, `-`, `*`, `/`, `{caret}`. Numeric operators can be u
 
 [WARNING]
 ====
-Because computers necessarily can only store approximations for most `DOUBLE` and `FLOAT` type values, it is not recommended to test these data types for exact equality or inequality. Instead, one should allow for an acceptable amount of error. The following example checks if `$0 = 5`, with an error of 0.00001 permitted:
+Because computers can only store approximations for most `DOUBLE` and `FLOAT` type values, testing these data types for exact equality or inequality is not recommended.
+Instead, one should allow for an acceptable amount of error.
+The following example checks if `$0 = 5`, with an error of 0.00001 permitted:
```

Comment:
Maybe instead of using =, since that's explicitly not what's happening, say "The following example checks whether `$0` is functionally equivalent to `5` by seeing if `$0` falls in a range of 5 plus or minus 0.0001.